### PR TITLE
Re-introduce merged data caching for QT

### PR DESCRIPTION
**Pull request by ehoogerbeets**
- other platforms can use it as well, but only QT does it by default

### Checklist

* [x] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [ ] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

**Comment by gouniLee** [comment]
if the `ilib._cacheMerged` value is set properly,  performance is improved with your change.  but still worse than before. When the fist creating the object, it takes much time than before. 
I noticed the line below is changed too,
https://github.com/iLib-js/iLib/blob/development/js/lib/DateFmt.js#L435
Previously,  it was 
`DateFactory._dynLoadDate(this.calName); `
it looks like you've changed to webpack support. Could we improve this part too?

File: js/lib/Utils.js [hunk 1]
```diff
@@ -384,6 +384,8 @@ Utils.loadData = function(params) {
         throw "Utils.loadData called without a callback. It must have a callback to work.";
     }
 
+    if (ilib._getPlatform() === "qt") ilib._cacheMerged = true;
+
```
Comment (ehoogerbeets) [inline js/lib/Utils.js]: All direct manipulation of the ilib name space should be contained within the ilib.js file as much as possible.

File: js/lib/ilib.js [hunk 2]
```diff
@@ -794,6 +794,6 @@ ilib.isDynData = function() {
  * merging is too slow, so the already-merged data is cached as well after the first time it
  * is requested. In this way, we sacrifice the memory footprint for the sake of speed.
  */
-if (ilib._getPlatform() !== 'qt') ilib._cacheMerged = false;
+ilib._cacheMerged = false;
```
Comment (ehoogerbeets) [inline js/lib/ilib.js]: Should be `ilib._cacheMerged = (ilib._getPlatform() === 'qt');`

Comment (gouniLee) [reply js/lib/ilib.js]: yeah..but If I add this line `ilib.js`. It makes lots of test case failures.

**Pull request merged**